src/lib/services/group-service.ts

```diff
@@ -30,6 +30,10 @@ import type { IUser } from '../types/user';
 import type EventService from '../features/events/event-service';
 import { SSO_SYNC_USER } from '../db/group-store';
 
+const setsAreEqual = (firstSet, secondSet) =>
```

Comment:
This is ugly and bloats the call site where it's actually relevant but only really useful in one place so I've pulled it out into a non exported function